### DISCUSSION TITLE
Pickl Error and Invalid Key every time load custom model from huggingface

### DISCUSSION THREAD

**Discussion by benagastov** [acheong08/Diffusion-ColabUI#43] [Q&A] [answered]
My colab shows me an error everytime I load custom model from huggingface like this one:
`During handling of the above exception, another exception occurred:

Traceback (most recent call last):
File "/content/stable-diffusion-webui/modules/safe.py", line 131, in load_with_extra
check_pt(filename, extra_handler)
File "/content/stable-diffusion-webui/modules/safe.py", line 98, in check_pt
unpickler.load()
_pickle.UnpicklingError: invalid load key, '<'.

-----> !!!! The file is most likely corrupted !!!! <-----
You can skip this check with --disable-safe-unpickle commandline argument, but that is not going to help you.

Traceback (most recent call last):
File "launch.py", line 251, in
start()
File "launch.py", line 246, in start
webui.webui()
File "/content/stable-diffusion-webui/webui.py", line 150, in webui
initialize()
File "/content/stable-diffusion-webui/webui.py", line 85, in initialize
modules.sd_models.load_model()
File "/content/stable-diffusion-webui/modules/sd_models.py", line 248, in load_model
load_model_weights(sd_model, checkpoint_info)
File "/content/stable-diffusion-webui/modules/sd_models.py", line 177, in load_model_weights
if "global_step" in pl_sd:
TypeError: argument of type 'NoneType' is not iterable`

I read some code in safe.py from here: https://github.com/AUTOMATIC1111/stable-diffusion-webui/blob/master/modules/safe.py

I already use `--disable-safe-unpickle` but still not working. Sometimes I want to try cool model like Ghibli studio ckpt

**Comment by acheong08** [comment, marked as answer]
Please use a direct download link. 

For example: 
https://huggingface.co/ShadoWxShinigamI/MidJourney-PaperCut 
should be 
https://huggingface.co/ShadoWxShinigamI/MidJourney-PaperCut/resolve/main/Mdjrny-pprct_step_7000.ckpt

**Reply by acheong08** [reply]
It has to be resolved via LFS

**Reply by acheong08** [reply]
I added some blaring notices for people like you. Please check issues before opening another one. This question has been answered countless times.

**Reply by acheong08** [reply]
That is not a direct link

**Reply by acheong08** [reply]
https://huggingface.co/iwannaaoe/ATitanBerryMix/resolve/main/ATitanStrawBerry.ckpt
This is the link you want

**Reply by benagastov** [reply]
Please remove this discussion, thank you. I think this one is shameful for me because I use wrong link and yet broken model compare to generated hands in Anythingv3
